core/trino-spi/src/main/java/io/trino/spi/type/LongTimestampType.java

```diff
@@ -148,6 +153,13 @@ private static int getFraction(Block block, int position)
         return block.getInt(position, SIZE_OF_LONG);
     }
 
+    @Override
+    public Optional<Range> getRange()
```

Comment:
would it make sense to precompute that in constructor and just return cached value here?

Reply:
moved to ctor